apps/internal/base/base.go

```diff
@@ -109,7 +110,7 @@ func AuthResultFromStorage(storageTokenResponse storage.TokenResponse) (AuthResu
 			return AuthResult{}, fmt.Errorf("problem decoding JWT token: %w", err)
 		}
 	}
-	return AuthResult{account, idToken, accessToken, storageTokenResponse.AccessToken.ExpiresOn.T, grantedScopes, nil}, nil
+	return AuthResult{account, idToken, accessToken, storageTokenResponse.AccessToken.ExpiresOn.T, grantedScopes, nil, true}, nil
```

Comment:
We should use named field syntax to prevent positioning bugs i.e. 
```go
return AuthResult{
    Account: account,
    ...
}
```

Reply:
Updated this!